test-data/genops-strip-asserts.test

```diff
@@ -0,0 +1,17 @@
+[case testStripAssert1]
+def g():
+  x = 1 + 2
+  assert x < 5
+  return x
+[out]
+def g():
+    r0, r1 :: short_int
+    r2 :: int
+    r3, x :: object
+L0:
+    r0 = 1
+    r1 = 2
+    r2 = r0 + r1 :: int
+    r3 = box(int, r2)
+    x = r3
+    return x
```

Comment:
```suggestion
    return x
    
```

Reply:
I am not sure if GitHub suggestions will work in this case, there is a missing newline at the end of file.